pkg/webhook/config.go

```diff
@@ -58,4 +58,9 @@ const (
 
```

Comment:
tested it and it works, code also looks good to me, but as it is still WIP according to title maybe something will still change, I guess?

Reply:
same- also deployed on my cluster- looking good to me!

Reply:
IMHO no more changes needed